tests/e2e/specs/add-paid-campaigns/add-paid-campaigns.test.js

```diff
@@ -370,169 +371,57 @@ test.describe( 'Set up Ads account', () => {
-	} );
-
-	test.describe( 'Set up billing', () => {
-		test.describe( 'Billing status is not approved', () => {
-			test.beforeAll( async () => {
-				await setupBudgetPage.fulfillBillingStatusRequest( {
-					status: 'pending',
-				} );
-			} );
-			test( 'It should say that the billing is not setup', async () => {
-				await page.getByRole( 'button', { name: 'Continue' } ).click();
-				await page.waitForLoadState( LOAD_STATE.DOM_CONTENT_LOADED );
-
-				await expect(
-					page.getByRole( 'button', {
-						name: 'Set up billing',
-						exact: true,
-					} )
-				).toBeEnabled();
-
-				await expect(
-					page.getByText(
-						'In order to launch your paid campaign, your billing information is required. You will be billed directly by Google and only pay when someone clicks on your ad.'
-					)
-				).toBeVisible();
-
-				await expect(
-					page.getByRole( 'link', {
-						name: 'click here instead',
-					} )
-				).toBeVisible();
-			} );
-
-			// eslint-disable-next-line jest/expect-expect
-			test( 'should open a popup when clicking set up billing button', async () => {
-				await checkBillingAdsPopup( page );
-			} );
-		} );
-
-		test.describe( 'Billing status is approved', async () => {
-			test.beforeAll( async () => {
-				await setupBudgetPage.fulfillBillingStatusRequest( {
-					status: 'approved',
-				} );
-
-				await setupAdsAccounts.mockAdsAccountsResponse( {
-					id: ADS_ACCOUNTS[ 1 ],
-					billing_url: null,
-				} );
-
-				// Simulate a bit of delay when creating the Ads campaign so we have enough time to test the content in the page before the redirect.
-				await page.route(
-					/\/wc\/gla\/ads\/campaigns\b/,
-					async ( route ) => {
-						await new Promise( ( f ) => setTimeout( f, 500 ) );
-						await route.continue();
-					}
-				);
-			} );
-			test( 'It should say that the billing is setup', async () => {
-				//Every 30s the page will check if the billing status is approved and it will trigger the campaign creation.
-				await setupBudgetPage.awaitForBillingStatusRequest();
-				await setupBudgetPage.mockCampaignCreationAndAdsSetupCompletion(
-					budget,
-					[ 'US' ]
-				);
-
-				await expect(
-					page.getByText(
-						'Great! You already have billing information saved for this'
-					)
-				).toBeVisible();
-
-				//It should redirect to the dashboard page
-				await page.waitForURL(
-					'/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fdashboard&guide=campaign-creation-success',
-					{
-						timeout: 30000,
-						waitUntil: LOAD_STATE.DOM_CONTENT_LOADED,
-					}
-				);
-			} );
 
-			test( 'It should show the campaign creation success message', async () => {
-				await expect(
-					page.getByRole( 'heading', {
-						name: "You've set up a paid Performance Max Campaign!",
-					} )
-				).toBeVisible();
-
-				await expect(
-					page.getByRole( 'button', {
-						name: 'Create another campaign',
-					} )
-				).toBeEnabled();
+		test( 'Paid Campaign can be created', async () => {
```

Comment:
@kt-12 This test has no assertion.

Reply:
I have moved content of this to the next test. As paid campaign created can be determined by success message.